scripts/merge-and-build.sh

```diff
@@ -74,35 +74,6 @@ if [ "${OSE_VERSION}" != "3.2" ] ; then
   ssh-add ${HOME}/.ssh/id_rsa
 fi # End check if we are version 3.2
 
-function sanity_check() {
```

Comment:
This will require to break up your python script and I don't think it's required now. If you don't have any other comments, I would like to move forward with this (it's still on the test branch) and test the flow.